cvat/apps/engine/views.py

```diff
@@ -928,11 +1128,58 @@ def upload_finished(self, request):
                         status=status.HTTP_400_BAD_REQUEST)
 
     @extend_schema(methods=['POST'],
-        summary='Method permanently attaches images or video to a task. Supports tus uploads, see more https://tus.io/',
+        summary="Method permanently attaches data (images, video etc.) to a task",
+        description=textwrap.dedent("""\
+            Allows to upload data to a task.
+            Supports the TUS open file uploading protocol (https://tus.io/).
+
+            Implements the following protocols:
+            a. A single Data request
+            b.1. An Upload-Start request
+            b.2.a. The regular TUS protocol requests (Upload-Length + Chunks)
+            b.2.b. Upload-Multiple requests
+            b.3. An Upload-Finished request
+
+            Requests:
+            - Data - POST, no extra headers or 'Upload-Start' + 'Upload-Finish' headers.
+              Contains Data in the body.
+            - Upload-Start - POST, has an 'Upload-Start' header.
+              Can contain upload metainfo in the body.
+            - Upload-Length - HEAD, has an 'Upload-Length' header (read the TUS protocol)
+            - Chunk - PATCH (read the TUS protocol).
+            - Upload-Finish - POST, has an 'Upload-Finish' header. Can contain Data in the body.
+            - Upload-Multiple - POST, has a 'Upload-Multiple' header. Contains Data in the body.
+
+            The 'Upload-Start' request allows to specify file order for TUS requests.
+            This may be needed because files in these requests and the requests themselves
+            can come (or be sent) unordered. To state that the input files are sent ordered,
+            pass an empty list of files in the 'files' field. If the files are sent unordered,
+            the ordered file list is expected in the 'files' field. It must be a list of string
+            file names, relatively to the dataset root.
+            Example:
+            files = [
+                cats/cat_1.jpg,
+                dogs/dog2.jpg,
+                image_3.png,
+                ...
+            ]
```

Comment:
Shouldn't it be something like:

```json
{
    "files": [
        "cats/cat_1.jpg",
        "dogs/dog2.jpg",
        "image_3.png",
        ...
    ]
}
```

instead? We're talking about the request body here, which is presumably going to be in JSON.

Reply:
Ok, done